templates/admin/emails/list.tmpl

```diff
@@ -59,6 +60,11 @@
 									{{svg (Iif .IsActivated "octicon-check" "octicon-x")}}
 								{{end}}
 							</td>
+							<td>
+								<div class="tw-flex tw-gap-2">
+									<a class="delete-button" href="" data-url="{{$.Link}}/delete" data-id="{{.ID}}" data-data-uid="{{.UID}}" data-data-email="{{.Email}}">{{svg "octicon-trash"}}</a>
```

Comment:
Maybe:
```suggestion
									<a class="delete-button" href="" data-url="{{$.Link}}/delete" data-id="{{.ID}}" data-data-uid="{{.UID}}">{{svg "octicon-trash"}}</a>
```
since the `email` has not be used?